app/views/content/landing/still-time-to-apply/_content.html.erb

```diff
@@ -0,0 +1,62 @@
+<div class="row">
+  <div class="grey col">
+    <section class="col col-720">
+      <p class="inset">
+        Every day, people are being accepted onto teacher training courses starting this September. Places fill up quickly at this time of year so don’t miss your opportunity to join a course while there’s still time to apply.
+      </p>
+<div class="row">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Find a teacher training course</h2>
+      <p>
+      Teacher training courses usually take 9 months full-time, or 18 to 24 months part time.
+      </p>
+      <p>
+      Most courses involve school placements with some theoretical learning.
+      </p>
+      <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find postgraduate teacher training.</a>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Get help to apply on time</h2>
+
+    <div class="inset">
+      <p>
+        Our teacher training advisers can save you time on research and guide you through everything you need to know to apply for teacher training.
+      </p>
+
+      <p>They can help you with:</p>
+
+      <ul>
+        <li>creating a personal statement that shows off your strengths</li>
+        <li>preparing for interviews</li>
+      </ul>
+
+      <p><%= link_to("Get a teacher training adviser", page_path("teacher-training-advisers")) %>.</p>
+    </div>
+  </section>
+
+    <section class="col col-720">
+
+    <h2 class="heading--box-blue">Applications from non-UK citizens</h2>
```

Comment:
I appreciate that this heading changed in response to my comment in the doc but I'm not sure it sits quite right - the other headings are more active and directed at the user. Worth changing to something like 'If you are a non-UK citizen'?

@philspray @EllieNodder